worlds/musedash/MuseDashCollection.py

```diff
@@ -46,6 +46,8 @@ class MuseDashCollections:
         "CHAOS Glitch",
         "FM 17314 SUGAR RADIO",
         "Yume Ou Mono Yo Secret",
+        "Echo over you... Secret",
+        "Tsukuyomi Ni Naru Replaced"
```

Comment:
```suggestion
        "Tsukuyomi Ni Naru Replaced",
```
There was a trailing comma before, and there is in all the other nearby lists.